dev-cpp/yaml_cpp/yaml_cpp0.7-0.7.0.recipe

```diff
@@ -17,19 +17,14 @@ libVersion="$portVersion"
 libVersionCompat="$libVersion compat >= ${libVersion%.*}"
 
 PROVIDES="
-	yaml_cpp$secondaryArchSuffix = $portVersion
+	yaml_cpp0.7$secondaryArchSuffix = $portVersion
 	lib:libyaml_cpp$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
 	"
-
-PROVIDES_devel="
-	yaml_cpp${secondaryArchSuffix}_devel = $portVersion
-	devel:libyaml_cpp$secondaryArchSuffix = $libVersionCompat
-	"
-REQUIRES_devel="
-	yaml_cpp$secondaryArchSuffix == $portVersion base
+REPLACES="
+	yaml_cpp$secondaryArchSuffix
```

Comment:
This should trigger the removal of the yaml_cpp_devel package. If you could confirm, then that's OK

Reply:
In Dutch, but should show it's working as expected :)
"Installeer" = "Install"
"verwijder" = "remove"
![yaml_cpp](https://github.com/haikuports/haikuports/assets/16057090/9de2626d-4978-453f-a7a9-f491dba11c98)